README.md

```diff
@@ -1,2 +1,61 @@
 # kddeployqt
-Deployment tool for Qt applications, similar to (android/mac/win)deployqt
+Deployment tool for Qt applications, similar to (android/mac/win)deployqt.
+
+The main differences between this tool and the existing ones([linuxdeploqt](https://github.com/probonopd/linuxdeployqt), [linuxdeploy](https://github.com/linuxdeploy/linuxdeploy), [linuxdeploy](https://github.com/probonopd/linuxdeploy), etc.) are:
+
+ - currently only linux host is supported.
```

Comment:
This doesn't seem like a main difference with other tools called linuxdeploysomething, obviously linux only too.

At least move it last in your list, so that it's less obvious that you're not really listing the differences with the other tools, but more describing this one ;)

An real list of differences (maybe with a features table) would of course be more interesting to the reader, trying to find out which tool to use.